README.md

```diff
@@ -24,9 +24,39 @@ sharing of data, prevents the "bitrot" of datasets wherein details of the
 capture are lost over time, and makes it possible for different tools to operate
 on the same dataset, thus enabling data portability between tools and workflows.
 
-(Taken from the
-[Introduction](https://github.com/gnuradio/SigMF/blob/master/sigmf-spec.md#introduction)
-of the specification document.)
+SigMF signal recordings typically involve a data file (e.g. a binary file of IQ
+or RF samples) and a metadata file containing plain text that describes the data.
+Together these files represent one recording, such as `example.sigmf-data` and
+`example.sigmf-meta`.  Here is a minimal example of a SigMF `.sigmf-meta` file:
+
+```yaml
+{
+    "global": {
+        "core:datatype": "cf32_le"
+        "core:sample_rate": 1000000,
+        "core:hw": "PlutoSDR with 915 MHz whip antenna",
+        "core:author": "Art Vandelay",
+        "core:version": "1.0.0",
+    },
+    "captures": [
+        {
+            "core:sample_start": 0
+            "core:frequency": 915000000,
+        }
+    ],
+    "annotations": []
+}
+```
+
+## Using SigMF
+
+There are four community-supported projects for using SigMF:
```

Comment:
Hmm -- so actually, there are currently just 3 community-supported projects.  Maybe replace "four" with "several" and move "4." to a new paragraph that explains that it's *so* easy to parse the file yourself, so "manually" is a really good option too.

Reply:
what do you think of the change i just made

Reply:
I think it's great.